test/openAPI/features/wallet_link_transaction.feature

```diff
@@ -22,3 +22,14 @@ Feature: The endpoint to validates the link-code and its expiry and generates th
     And The /linked-authorization/link-transaction endpoint response should have content-type: application/json header
     And The /linked-authorization/link-transaction endpoint response should match json schema
     And The /linked-authorization/link-transaction response should contain errorCode property equals to "invalid_link_code"
+
+  @negative
+  Scenario: Not able to validate the link-code and its expiry and generate the linkTransactionId because of expired linkCode
```

Comment:
For expired linkcode, we have to generate a link code using /link-code wait for 30 seconds for it to expire and then try to use it for /link-transaction to get back a invalid_link_code error code

Reply:
In the linked-authorization/link-code response there is "expiredDateTime" and there is no information about this "expiredDateTime" should be only 30s longest than responseTime. I'm not sure if software's candidates will now about it. 
But I will add this test case ;)

Reply:
You are right, this value can be configurable, but for now 30 seconds is good enough for the use case. We can also improve API document about the assumptions.